Resources/Prototypes/Maps/kettle.yml

```diff
@@ -1,46 +1,47 @@
 - type: gameMap
-  id: Meta
-  mapName: 'Meta Station'
-  mapPath: /Maps/meta.yml
+  id: Kettle
+  mapName: 'Kettle'
+  mapPath: /Maps/kettle.yml
   minPlayers: 35
   stations:
-    Meta:
-      mapNameTemplate: '{0} Meta Station {1}'
+    Kettle:
+      mapNameTemplate: '{0} Kettle {1}'
       nameGenerator:
         !type:NanotrasenNameGenerator
-        prefixCreator: 'TG'
+        prefixCreator: '14'
       overflowJobs:
         - Passenger
       availableJobs:
         CargoTechnician: [ 3, 3 ]
         Passenger: [ -1, -1 ]
-        Bartender: [ 2, 2 ]
+        Bartender: [ 1, 2 ]
         Botanist: [ 3, 3 ]
         Chef: [ 2, 2 ]
-        Clown: [ 1, 1 ]
-        Janitor: [ 2, 2 ]
-        Mime: [ 1, 1 ]
+        Clown: [ 1, 2 ]
+        Janitor: [ 3, 3 ]
+        Mime: [ 1, 2 ]
         Captain: [ 1, 1 ]
         HeadOfPersonnel: [ 1, 1 ]
         ChiefEngineer: [ 1, 1 ]
-        StationEngineer: [ 4, 4 ]
+        StationEngineer: [ 4, 6 ]
         ChiefMedicalOfficer: [ 1, 1 ]
-        MedicalDoctor: [ 3, 3 ]
+        MedicalDoctor: [ 3, 4 ]
         Chemist: [ 2, 3 ]
         ResearchDirector: [ 1, 1 ]
         Scientist: [ 4, 4 ]
         HeadOfSecurity: [ 1, 1 ]
-        SecurityOfficer: [ 4, 4 ]
+        SecurityOfficer: [ 4, 6 ]
         Chaplain: [ 1, 1 ]
         Warden: [ 1, 1 ]
         Librarian: [ 1, 1 ]
         Lawyer: [ 2, 2 ]
         Quartermaster: [ 1, 1 ]
-        SalvageSpecialist: [ 3, 3 ]
-        Musician: [ 1, 1 ]
+        SalvageSpecialist: [ 3, 4 ]
+        Musician: [ 1, 2 ]
         AtmosphericTechnician: [ 3, 3 ]
         TechnicalAssistant: [ 2, 2 ]
         MedicalIntern: [ 2, 2 ]
         ServiceWorker: [ 2, 2 ]
-        SecurityCadet: [ 2, 2 ]
+        SecurityCadet: [ 3, 4 ]
+        Reporter: [ 1, 1 ]
```

Comment:
this job belongs to bagel and you should prolly figure out another one (or just remove it for the time being)